tools/scripts/deploy.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2019 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+rootdir="${scriptdir%/*/*}"
+
+dbg() {
+    [ "$VERBOSE" = "true" ] && echo "$@"
+}
+
+err() {
+    printf '\033[1;31m'"$@\n"'\033[0m'
+}
+
+success() {
+    printf '\033[1;32m'"$@\n"'\033[0m'
+}
+
+run() {
+    dbg "$*"
+    "$@" || exit $?
+}
+
+
+usage() {
+    echo "usage: $(basename $0) [-hvsmdCD]"
+}
+
+main() {
+    OPTS=`getopt -o 'hvdb:i:' --long verbose,help,deploy,build-dir:,target-ip:,target-user:,target-pass:,proxy-ip:,proxy-user:,proxy-pass: -n 'parse-options' -- "$@"`
+
+    if [ $? != 0 ] ; then err "Failed parsing options." >&2 ; usage; exit 1 ; fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -v | --verbose)       VERBOSE=true; shift ;;
+            -h | --help)          usage; exit 0; shift ;;
+            -d | --deploy)        DEPLOY=true; shift ;;
+            -b | --build-dir)     BUILD_DIR="$2"; shift; shift ;;
+            --proxy-ip)           PROXY_IP="$2"; shift; shift ;;
+            --proxy-user)         PROXY_USER="$2"; shift; shift ;;
+            --proxy-pass)         PROXY_PASS="$2"; shift; shift ;;
+            --target-ip)          TARGET_IP="$2"; shift; shift ;;
+            --target-user)        TARGET_USER="$2"; shift; shift ;;
+            --target-pass)        TARGET_PASS="$2"; shift; shift ;;
+            -- ) shift; break ;;
+            * ) err "unsupported argument $1"; usage; exit 1 ;;
+        esac
+    done
+
+    echo "Start"
+    tmpdir=$(mktemp -d)
+    mkdir -p ${tmpdir}/opt/prplmesh
+    mkdir -p ${tmpdir}/usr
+
+    cp -r ${BUILD_DIR}/out/* ${tmpdir}/opt/prplmesh
```

Comment:
AFAIR the correct files are placed on the install dir, and on the "out" dir the files are irrelevant. Did we fix it?

Reply:
The only reason why the out dir isn't removed yet is because supposedly there were scripts like this one that still used it. I'd be more than happy to remove it!